libraries/AP_Airspeed/AP_Airspeed_UAVCAN.cpp

```diff
@@ -0,0 +1,176 @@
+#include <AP_HAL/AP_HAL.h>
+
+#if HAL_WITH_UAVCAN
+
+#include "AP_Airspeed_UAVCAN.h"
+
+#include <AP_BoardConfig/AP_BoardConfig_CAN.h>
+#include <AP_UAVCAN/AP_UAVCAN.h>
+
+#include <uavcan/equipment/air_data/RawAirData.hpp>
+
+extern const AP_HAL::HAL& hal;
+
+#define debug_airspeed_uavcan(level_debug, can_driver, fmt, args...) do { if ((level_debug) <= AP::can().get_debug_level_driver(can_driver)) { printf(fmt, ##args); }} while (0)
+
+// UAVCAN Frontend Registry Binder
+UC_REGISTRY_BINDER(AirspeedCb, uavcan::equipment::air_data::RawAirData);
+
+AP_Airspeed_UAVCAN::DetectedModules AP_Airspeed_UAVCAN::_detected_modules[] = {0};
+AP_HAL::Semaphore* AP_Airspeed_UAVCAN::_sem_registry = nullptr;
+
+// constructor
+AP_Airspeed_UAVCAN::AP_Airspeed_UAVCAN(AP_Airspeed &airspeed, uint8_t instance) :
```

Comment:
use _frontend and _instance for variable names instead. "instance" variable name is shadowing Backends member with same name, generating error in CI.

Reply:
solved.